ocs_ci/helpers/managed_services.py

```diff
@@ -233,3 +233,24 @@ def verify_osd_used_capacity_greater_than_expected(expected_used_capacity):
             )
             return True
     return False
+
+
+def verify_osds_are_on_correct_machinepool():
+    """
+    Verify that the OSD pods are running on nodes which are part of the correct machinepool.
+    Applicable for Managed Services.
+
+    """
+    osd_nodes = get_osd_running_nodes()
+    osd_node_objs = get_node_objs(osd_nodes)
+    for node_obj in osd_node_objs:
+        annotation = (
+            node_obj.get()
+            .get("metadata")
+            .get("annotations")
+            .get("machine.openshift.io/machine")
+        )
+        assert (
+            "ceph-osd-nodepool" in annotation
```

Comment:
I would change this line to `{constants.OSD_NODE_POOL} in annotation`.

Reply:
Sorry, I meant to replace it with the line `constants.OSD_NODE_POOL in annotation`

Reply:
Done